### PR TITLE
include smss case

### PR DESCRIPTION
**Pull request by vera-chan**
Signed-off-by: Vera <vera.xx.c@me.com>
Updates to READMEs to include the SMSS case

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,10 +55,9 @@ The plugin requires Gradle version 5 features and will not work correctly on ear
...
- * You have a CICS region that is at CICS® TS V5.6 or later
- * This region is configured to be a WUI region for the CICSplex that contains the deployment target region
- * This WUI region is configured to use the CMCI JVM server, including the CICS bundle deployment API
+However, if you're using the `deployCICSBundle` task of the plugin to deploy bundles to CICS, you must enable the CICS bundle deployment API. The CICS bundle deployment API is supported by the CMCI JVM server that must be set up in a WUI region or a single region to turn it into an SMSS. See the [CICS TS doc](https://www.ibm.com/docs/en/cics-ts/6.1_beta?topic=suc-configuring-cmci-jvm-server-cics-bundle-deployment-api) for details. To use the `deployCICSBundle` task, make sure that:
```
Comment (tom-foyle) [inline README.md:58]: I'm not sure that "to turn it into an SMSS" is necessary here. It's possible that the region is already an SMSS if it's using "traditional" CMCI but not CMCI JVM server.

**Comment by vera-chan** [comment]
@tom-foyle Hi Tom, I just pushed another commit to this PR (d0fbac9083d3f9a466e7af21af1db7e5e39b0a6d) to address your comment. Could you take a look. Thanks.

**Comment by vera-chan** [comment]
Force pushed again with signed-off commit

**Review by tom-foyle — APPROVED** [review]

**Comment by vera-chan** [comment]
Close and reopen the PR after the Analyze (java) test is re-enabled.

**Pull request closed without merging**